persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/AkkaTaggerAdapter.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.scaladsl.persistence
+
+import akka.cluster.sharding.typed.scaladsl.EntityContext
+
+object AkkaTaggerAdapter {
+
+  /**
+   * Adapts an existing Lagom [[AggregateEventTagger]] to a
+   * function {{{Event => Set[String]}}} as expected by Akka Persistence Typed {{{EventSourcedBehavior.withTagger}}} API.
+   */
+  def fromLagom[Event <: AggregateEvent[Event]](
+      entityCtx: EntityContext,
+      lagomTagger: AggregateEventTagger[Event]
```

Comment:
Note that I'm passing the EntityContext. In order to make sure we will tag correctly, we should ask for the context and use the entityId. If it was a String, user could pass anything or even PersistenceId.toString which would be wrong.